src/main/zapHomeFiles/hudtutorial/en_GB/Warning.html

```diff
@@ -12,7 +12,7 @@ <h1 class="warning">Obligatory Warning</H1>
 	You should only use ZAP (and therefore also the HUD) on applications that you either
 	own or have permission to attack.
 	<p>
-	The good news is that if you installed ZAP on your own machine then you do own this tutorial,
+	The good news is that if you installed ZAP on your own machine then you own this tutorial,
```

Comment:
happy to drop this.

It was a grammarly suggestion (it really doesn't like certain extra words/longer phrases).